### ISSUE TITLE
can boot into the guest successfully,cannot get back to ubantu,get a no signal

### ISSUE THREAD

**Issue by Jotalz** [QaidVoid/Complete-Single-GPU-Passthrough#36]
I added the logging instructions according to the problem document
```
exec 19>/home/user/Desktop/stoplogfile
BASH_XTRACEFD=19
set -x
```
And the generated log is as follows:
`+ sleep 10
+ modprobe -r vfio_pci
+ modprobe -r vfio
+ modprobe -r vfio_iommu_type1
+ modprobe -r vfio_virqfd
+ virsh nodedev-reattach pci_0000_08_00_0
+ virsh nodedev-reattach pci_0000_08_00_1
+ echo 1
+ echo 1
+ echo efi-framebuffer.0
+ modprobe nvidia
+ modprobe nvidia_modeset
+ modprobe nvidia_uvm
+ modprobe nvidia_drm
+ modprobe drm_kms_helper
+ modprobe i2c_nvidia_gpu
+ modprobe drm
+ sleep 10
+ systemctl start gdm3`

And my stop.sh is:
```
sleep 10

modprobe -r vfio_pci
modprobe -r vfio
modprobe -r vfio_iommu_type1
modprobe -r vfio_virqfd

virsh nodedev-reattach pci_0000_08_00_0
virsh nodedev-reattach pci_0000_08_00_1
echo 1 > /etc/class/vtconsole/vtcon0/bind
echo 1 > /etc/class/vtconsole/vtcon1/bind

echo efi-framebuffer.0 > /sys/bus/platform/drivers/efi-framebuffer/bind

modprobe nvidia
modprobe nvidia_modeset
modprobe nvidia_uvm
modprobe nvidia_drm
modprobe drm_kms_helper
modprobe i2c_nvidia_gpu
modprobe drm

sleep 10

systemctl start gdm3
```
Does this mean something went wrong while binding?Can you guide me how to solve this problem,thinks!

**Comment by QaidVoid** [comment]
Have you tried starting `display-manager` service instead of gdm3?

**Comment by FORDLESS** [comment]
I tried adding `systemctl start display-manager` after `systemctl start gdm3` , but when i shutdown windows,the screen is still black.
The most terrible thing about this is that I don’t know how to locate the error point,
When I boot Ubuntu from power off, I can see several errors about the GPU, such as `Failed to start NVIDIA persistence Daemon.`
Although the system can boots normally, I'm not sure if this will cause the KVM to not return host.
do you have any ideas?

**Comment by FORDLESS** [comment]
I found this question under most Single-GPU-Passthrough posts. I tried most of the methods they mentioned but it didn't work.
But I can't do other methoned like that, lower the driver version to 460 because myGPU is 4060ti and there is no such early version.I think this is indeed a more difficult problem to solve

**Comment by FORDLESS** [comment]
> https://forums.developer.nvidia.com/t/failed-to-start-nvidia-persistence-daemon-on-ubuntu-20-04/160985
> 
> I guess this might help.
> 
> TLDR: Blacklist nvidiafb. On file `/etc/modprobe.d/blacklist.conf`, add the line `blacklist nvidiafb`

![ovmf4](https://github.com/QaidVoid/Complete-Single-GPU-Passthrough/assets/57427866/a9971bfe-4b18-4f88-ac32-c69ba501eb80)
Could this be the cause of cannot get back?

**Comment by QaidVoid** [comment]
AMD GPUs have issues with reset, but NVIDIA should have proper reset support..
